.mocharc.js

```diff
@@ -7,6 +7,8 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
+process.env.NODE_ENV = 'test';
```

Comment:
If the intention is to use `config/test.json`, then could `process.env.NODE_CONFIG_ENV = 'test'` be used instead?

Could this setting be moved to `Makefile` to avoid affecting other mocha tests (e.g. e2e tests) which already use their own `NODE_CONFIG_ENV`?

Reply:
> Intention was to stop tests to use `development.json` to read config.

:+1: then I'd encourage using [`NODE_CONFIG_ENV`][1], as it's less likely to have side effects than changing `NODE_ENV`.

[1]: https://github.com/getodk/central-backend/pull/1421#discussion_r1990089643

Reply:
(I see you've switched to `NODE_CONFIG_ENV` :+1:)